src/components/WalletAssets.vue

```diff
@@ -15,7 +15,6 @@
       divider
       :assets="formattedAccountAssets"
       :size="3"
-      @show-details="handleOpenAssetDetails"
```

Comment:
Could you remove from this component these methods please:
```
getAssetIconStyles
getAssetIconClasses
```
they are not used anymore